scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fetches and formats review comments from a GitHub Pull Request."""
+
+import argparse
+import os
+import sys
+import firebase_github # Assumes firebase_github.py is in the same directory or python path
+
+# Attempt to configure logging for firebase_github if absl is available
+try:
+    from absl import logging as absl_logging
+    # Set verbosity for absl logging if you want to see logs from firebase_github
+    # absl_logging.set_verbosity(absl_logging.INFO)
+except ImportError:
+    pass # firebase_github.py uses absl.logging.info, so this won't redirect.
+
+
+def print_contextual_diff_hunk(diff_hunk, comment_position, context_lines_count):
+    if not diff_hunk or not diff_hunk.strip():
+        print("(No diff hunk available or content is empty)")
+        return
+
+    hunk_lines = diff_hunk.split('\n') # Note: Python's split('\n') is generally fine.
+
+    # Case 1: User explicitly wants the full hunk
+    if context_lines_count == 0:
+        print(diff_hunk)
+        return
+
+    # Case 2: Contextual display is requested (context_lines_count > 0),
+    # but comment is not on a specific line or position is invalid for contextual display.
+    if comment_position is None or comment_position < 1 or comment_position > len(hunk_lines):
+        print("(Comment is not on a specific line in the diff, or position is invalid; full hunk context suppressed by --context-lines setting)")
```

Comment:
I think if hunk_position is None, instead of printing this comment, just print out the *last N lines* of the diff hunk, where N is the context-lines setting.